test/docs/testbed/README.testbed.Cli.md

```diff
@@ -0,0 +1,8 @@
+
+# run tests
+
+```
+docker run --network host -v $PWD:/dash --mount src=/etc/localtime,target=/etc/localtime,type=bind,readonly -it dash/keysight bash
```

Comment:
please contribute to sonic-mgmt and make dash testbed a part of sonic-mgmt test

Reply:
We absolutely plan to be located in the sonic-mgmt repo as DASH support becomes available.  In the meantime we would like to perform testing early / incubate.  Methodology and structure will be consistent w/sonic-mgmt to ensure smooth integration.